ntbs-service/wwwroot/source/Components/ValidateTravelOrVisit.ts

```diff
@@ -1,18 +1,18 @@
 import Vue from 'vue';
-import {getHeaders} from '../helpers';
+import {convertStringToNullableInt, getHeaders} from '../helpers';
```

Comment:
How sure are you that this is the only affected endpoint?

Reply:
I have now changed ValidateContactTracing to also use this helper, as this had a different way of handling integers. After a sweep that is the only other place I found that needs to send integers explicitly